invisible_cities/reco/paolina_functions_test.py

```diff
@@ -239,20 +239,15 @@ def test_voxels_with_no_hits(ICDATADIR):
     size = 15.
     vox_size = np.array([size,size,size],dtype=np.float16)
 
-    with tb.open_file(hit_file, mode='r') as h5in:
-
-        h5extents = h5in.root.MC.extents
-        events_in_file = len(h5extents)
-
-        for i in range(events_in_file):
-            if h5extents[i]['evt_number'] == evt_number:
-                evt_line = i
-                break
-
-        hits_dict = load_mchits(hit_file, (evt_line, evt_line+1))
-        voxels = voxelize_hits(hits_dict[evt_number], vox_size, strict_voxel_size=False)
-        for v in voxels:
-            assert sum(h.E for h in v.hits) == v.E
+    hits_df  = load_mchits_df(hit_file)
+    ## Patch to BHits, temporary?
+    evt_hits = hits_df.loc[evt_number][['x', 'y', 'z', 'energy']].values
+    bhit_seq = [BHit(*hit) for hit in evt_hits]
```

Comment:
I think we should have an official function that casts mchits_df to a dictionary (the output of load_mchits) should be in mcinfo_io